lib/arjdbc/jdbc/adapter.rb

```diff
@@ -289,12 +289,14 @@ def active?
 
       # @override
       def reconnect!
+        super
```

Comment:
yeah it totally makes sense on first sight - wonder why it doesn't matter much (no issues reported).
no need to if you're busy with 5.x, maybe could you point me to the tests that were failing before?